.gitignore

```diff
@@ -121,3 +125,8 @@ bin/nimbus-fml.sh
 firefox-ios/nimbus-fml-configuration.local.sh
 firefox-ios/bin/nimbus-fml.sh
 firefox-ios/bin/nimbus-fml-configuration.local.sh
+
+# Lists
+Lists/disconnect-*
+focus-ios/shavar-*
+shavar-prod-lists
```

Comment:
Side note, it seems we have a `firefox-ios/shavar-prod-lists` and a `shavar-prod-lists` folder. I think we can improve this (in another task) as Focus iOS is fetching an older version of the Shavar-prod-lists than Firefox is doing. At least we could ensure that the Focus shavar-prod-lists is under the Focus folder.

I'll create a task to update this once this PR is merged

Reply:
Created https://mozilla-hub.atlassian.net/browse/FXIOS-8554 for this